docs/user-guide/fundamental_topics/interface_configuration.md

```diff
@@ -97,7 +97,7 @@ Below is a sample JSON configuration file with some of the more common options.
 - **`uninterruptible`** - Setting `uninterruptible` to `false` allows the federate to be interrupted if there is a signal available for it to receive. This is a [timing configuration](./timing_configuration.md) option.
 - **`terminate_on_error`** - By default, HELICS will not terminate execution of every participating federate if an error occurs in one. However, in most cases, if such an error occurs, the cosimulation is no longer valid. Setting `terminate_on_error` frees the federate from the broker if there is an error in execution, which simplifies debugging. This will prevent your federate from hanging in the event that another federate fails.
 - **`wait_for_current_time_update`** - There are times when HELICS will grant the same simulated time to a number of federates simultaneously. There is a possibility of this leading to unexpected co-simulation results if federates are unexpectedly operating on old data. Using this flag, HELICS uses this option to provide the ability for one federate to always be granted this time last, after all other federates that have been granted this time have requested a later time. This ensures that the federate with this flag set will have all the latest information from all other federates before it begins execution at the granted time.
-- **`federate_init_string`** - This option provides a way of passing in a large number of configuration options that a federate needs during initialization. You can consult the [Configuration Options Reference](../../references/configuration_options_reference.md#broker-init-string-brokerinitstring-brokerinitstring) page for a more complete list but there are a few worth bringing up specifically. `--broker_address=<IP address>` and `--port=<port number>`- Allows you to specify the IP address and port number of the broker to which you want this federate to connect. You can consult the [Advanced Topics section of the User Guide](../advanced_topics/advanced_topics_index.md) to see further explanation of how to handle more complex broker configuration.
+- **`federate_init_string`** - This option provides a way of passing in a large number of configuration options that a federate needs during initialization. You can consult the [Configuration Options Reference](../../references/configuration_options_reference.md#broker_init_string--brokerinitstring--brokerinitstring-) page for a more complete list but there are a few worth bringing up specifically. `--broker_address=<IP address>` and `--port=<port number>`- Allows you to specify the IP address and port number of the broker to which you want this federate to connect. You can consult the [Advanced Topics section of the User Guide](../advanced_topics/advanced_topics_index.md) to see further explanation of how to handle more complex broker configuration.
```

Comment:
Did you check if this reference works?  not sure about the `--`

Reply:
Yeah, [MyST provides a tool](https://myst-parser.readthedocs.io/en/stable/syntax/optional.html#syntax-header-anchors) that identifies the anchors it will create in an offline manner so you don't have to guess. 

`myst-anchors -l <int: heading level you want to make anchors for> path/to/markdown.md`